spring-cloud-dataflow-server-core/src/test/java/org/springframework/cloud/dataflow/server/controller/TaskExecutionControllerTests.java

```diff
@@ -374,6 +374,17 @@ void getAllThinExecutions() throws Exception {
 			.andExpect(jsonPath("$._embedded.taskExecutionThinResourceList", hasSize(4)));
 	}
 
+
+	@Test
```

Comment:
Why does the source test differ https://github.com/spring-cloud/spring-cloud-dataflow/pull/5994/files#diff-a37ece622810b7f245fa769ce9ee2df687e2c8f9ab859daa8fb8bbe04f74f16fR344 ?

Reply:
I need to merge the two sets of functionality.